docs/source/getting-started/quickstart.rst

```diff
@@ -165,8 +165,26 @@ Stop/terminate a cluster
 =========================
 
 When you are done, run :code:`sky stop mycluster` to stop the cluster. To
-terminate a cluster instead, run :code:`sky down mycluster`.  Find more commands that
-manage the lifecycle of clusters :ref:`here <interactive-nodes>`.
+terminate a cluster instead, run :code:`sky down mycluster`.
+
+Find more commands that manage the lifecycle of clusters in the :ref:`CLI reference <cli>`.
+
+Scaling out
+=========================
+
+So far, we have used the `serverful` CLIs of SkyPilot, where the notion of clusters is central.
+This is useful for development since it is convenient to log into a cluster and perform debugging.
```

Comment:
Isn't it also ~useful~ required for running jobs for on-demand clusters?

Reply:
Reworded.